src/tasks/runstart.ts

```diff
@@ -304,11 +304,11 @@ export const RunStartQuest: Quest = {
         get("instant_savePantogram", false),
       do: (): void => {
         Pantogram.makePants(
-          mainStatStr,
+          mainStatStr.toString(),
           "Hot Resistance: 2",
           "Maximum HP: 40",
           "Combat Rate: -5",
-          "Weapon Damage: 20"
+          "Spell Damage Percent: 20"
```

Comment:
If you want to update this, you need to ensure that the user is already capping Weapon Damage. This is not the case in the basic run plan, and the ability to double Weapon Damage with bow-legged swagger means that choosing Weapon Damage is a 0.8 turnsave while Spell Damage Percent is only a 0.4 turnsave.

Reply:
Understood; I'm not sure when I changed this. Updated back.